src/System.Management.Automation/engine/parser/tokenizer.cs

```diff
@@ -1229,23 +1229,125 @@ private int SkipBlockComment(int i)
             return i;
         }
 
-        private static char Backtick(char c)
+        private string Backtick(char c)
```

Comment:
> It looks like the only reason to check the length is to support doubling { and }

Actually it is used to determine whether the resulting string is a Unicode esc sequence that corresponds to a surrogate pair.  That is the **only** case where the string returned by `Backtick` contains more than one character.  

In the non-bracketed case e.g. `` "`u007b"`` you will never get back a surrogate pair.  `` "`u7b"`` is not a valid seq - has to be 4 hex digits in this C#/Bash style Unicode esc seq.

FWIW I didn't feel great about the changes to the six `Backtick` call sites.  I'm going to try something to at least consolidate the duplicate code in a single method to stay copasetic with the DRY principle.